### ISSUE TITLE
Fix remaining accessibility audit issues

### ISSUE THREAD

**Issue by brentryanjohnson** [ONRR/doi-extractives-data#4607]
## Flag 1

[id] attributes on the page are not unique
The value of an id attribute must be unique to prevent other instances from being overlooked by assistive technologies. 

Failing Elements
svg#Layer_1

## Flag 2

Form elements do not have associated labels
Labels ensure that form controls are announced properly by assistive technologies, like screen readers. 

Failing Elements
select
